app/views/layouts/application.html.erb

```diff
@@ -6,7 +6,6 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
-    <%= javascript_include_tag 'application' %>
```

Comment:
I guess webpacker does this now, ok

Reply:
In Rails 6, the new recommended thing is to use javascript packs instead of the asset pipeline for js.  This particular layout doesn't require any JS at the moment, so I removed the javascript_include_tag altogether.  If we end up needing js though, we can create an additional (separate) js pack for the application layout context.  The application layout is only used by non-react parts of Hyacinth.